testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +248,26 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+def test_PDB_elements():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_elements, format='PDB')
+    element_list = ['N', 'C', 'C', 'O', 'C', 'C', 'O', 'N', 'H', 'H', 'H',
+                    'H','H','H', 'H', 'H', 'CU', 'FE', 'C', 'MG']
+    assert len(u.atoms.elements) == len(element_list)
+    assert all(i == j for i,j in zip(element_list, u.atoms.elements))
+
+
+
+def test_PDB_elements_warnings():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_wrong_ele, format='PDB')
```

Comment:
Again, do we really not have a PDB file with missing elements? Surely one of your new files is redundant. Generally we try not to add more files to the test suite if an existing one suffices, to keep it as light as possible.

Reply:
using `StringIO` is a way of having a throwaway PDB file inlined in the code, I think there's examples in either PDB or GRO...

Reply:
That actually sounds like a better option, I will use that instead